src/challenges.js

```diff
@@ -22,6 +41,13 @@ const cacheFunction = cb => {
   // If the returned function is invoked with arguments that it has already seen
   // then it should return the cached result and not invoke `cb` again.
   // `cb` should only ever be invoked once for a given set of arguments.
+  const cache = {};
+ return (...args) => {
+       if (Object.prototype.hasOwnProperty.call(cache, ...args)) {
```

Comment:
Instead of `Object.prototype.hasOwnProperty.call()`, we can use the `in` operator:

```JS
const cacheFunction = (cb) => {
  const cache = {};
  return (arg) => {
    if (arg in cache) return cache[arg];
    return cache[arg] = cb(arg);
  };
};
```